Milliner/composer.json

```diff
@@ -31,7 +31,7 @@
         ],
         "test": [
             "@check",
-            "phpunit"
+	    "phpunit --coverage-html 'reports/clover_html' tests/"
```

Comment:
I'm guessing you were using this to check your test coverage and forgot to revert it before pushing. The PHPUnit stuff is defined in the `phpunit.xml.dist` so could you change this back.